rpc-jobs/jjb-setup.yml

```diff
@@ -11,17 +11,16 @@
         - bool:
             name: IGNORE_CACHE
             description: "Ignore cache when updating jobs."
-        - string:
-            name: JJB_REPO
-            default: https://github.com/rcbops/rpc-gating
-        - string:
-            name: JJB_BRANCH
-            default: master
+        - bool:
+            name: DELETE_OLD
+            description: "Remove jobs that are no longer defined."
+            default: false
```

Comment:
Personally I would think that you'd want this to default to 'true' so that jobs are always cleaned up. This enforces that all jobs are always cleaned up unless you specifically don't want to do that.